zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -239,6 +239,8 @@ public Note cloneNote(String sourceNoteID, String newNoteName, AuthenticationInf
     Note newNote = createNote(subject);
     if (newNoteName != null) {
       newNote.setName(newNoteName);
+    } else {
+      newNote.setName("Note " + newNote.getId());
```

Comment:
@jongyoul I did the changes already with this ticket and running CI build now.Please let me know if I need to revert them.